docs/apps/geoconda.md

```diff
@@ -74,9 +79,10 @@ If you think that some important GIS package for Python is missing from here, yo
 
 ## Available
```

Comment:
```suggestion
## Availability
```

Reply:
Available is old heading used in all Apps pages, would not change it now.